evm/verify-contract.js

```diff
@@ -320,6 +330,46 @@ async function processCommand(config, chain, options) {
             break;
         }
 
+        case 'AxelarAmplifierGateway': {
+            let args = options.args;
+
+            if (args) {
+                args = JSON.parse(args);
+                options = { ...options, ...args };
+            }
+
+            const AxelarAmplifierGateway = require('@axelar-network/axelar-gmp-sdk-solidity/artifacts/contracts/gateway/AxelarAmplifierGateway.sol/AxelarAmplifierGateway.json');
+            const amplifierGatewayFactory = await getContractFactoryFromArtifact(AxelarAmplifierGateway, wallet);
+            const amplifierGateway = amplifierGatewayFactory.attach(options.address || chain.contracts.AxelarGateway.address);
+
+            const implementation = await amplifierGateway.implementation();
+            const previousSignersRetention = (await amplifierGateway.previousSignersRetention()).toNumber();
+            const domainSeparator = await amplifierGateway.domainSeparator();
+            const minimumRotationDelay = (await amplifierGateway.minimumRotationDelay()).toNumber();
+
+            options.keyID =
+                chain.contracts.AxelarGateway.startingKeyIDs[0] || options.keyID || `evm-${chain.axelarId.toLowerCase()}-genesis`;
```

Comment:
keyIDs aren't used for amplifier gateway when retrieving signers from axelar, see the deployment script